phpBB/styles/prosilver/theme/content.css

```diff
@@ -361,6 +361,12 @@ dd.option {
 	line-height: 1.4em;
 }
 
+.agreement-text {
+	font-size: 1.0em;
+	line-height: 1.4em;
+	margin-bottom: 1em;
+}
+
```

Comment:
So let me see, you are removing the P tag fromt the HTML and adding a class there, but in the meantime you are adding the P tag in the language files? That looks to me a bit weird.

Am I missing somehting?

Reply:
To add HTML in the PHP files IIRC is not something that should be done, if possible, BTW.

Reply:
I would normally agree, but `p` tags are more semantically appropriate in this case than `br` tags.

Reply:
Default CSS Settings

Most browsers will display the <p> element with the following default values:

```
p {
  display: block;
  margin-top: 1em;
  margin-bottom: 1em;
  margin-left: 0;
  margin-right: 0;
} 
```

But that's not the point really, unless I am reading this commit wrongly.

Reply:
I think you're focusing on how it looks, which has nothing to do with semantic meaning.
https://en.wikipedia.org/wiki/Semantic_HTML